docs/log-correlation.asciidoc

```diff
@@ -41,6 +43,7 @@ In other scenarios, like for unstructured logs,
 you'll need to add APM identifiers to your logs in any easy to parse manner.
 
 Log correlation relies on these fields:
+
```

Comment:
[for reviewer] I initially wanted to only fix this small missing line to properly display the list items below.